micro_dl/config_inference_dataset.yml

```diff
@@ -0,0 +1,69 @@
+dataset:
+  augmentation: false
+  data_dir: /hpc/projects/comp_micro/projects/virtualstaining/torch_microDL/data/2022_09_07_Soorya_KidneyRickettsia/data/
+  input_channels:
+  - 3
+  mask_channels:
+  - 4
+  model_task: regression
+  normalize: false
+  random_seed: 42
+  split_by_column: pos_idx
+  split_ratio:
+    test: 0.05
+    train: 0.05
+    val: 0.9
+  target_channels:
+  - 0
+  train_fraction: 1
+network:
+  activation:
+    type: relu
+  batch_norm: true
+  block_sequence: conv-activation-bn
+  class: UNetStackTo2D
+  data_format: channels_first
+  depth: 5
+  dropout: 0.2
+  filter_size: 3
+  final_activation: linear
+  height: 2048
+  num_convs_per_block: 2
+  num_filters_per_block:
+  - 16
+  - 32
+  - 64
+  - 128
+  - 256
+  num_input_channels: 1
```

Comment:
Since we already have specified which input and output channels we use, it seems a bit redundant to again specify how many of each there are, it's basically just len(input_channels) etc.
I know this is not your design but a remnant of bad design from way back when, but still it would be nice to remove these.